app/models/contact_message.rb

```diff
@@ -0,0 +1,11 @@
+class ContactMessage < ApplicationRecord
+  # Callabcks
+  # Associations
+  # Enumerator
+  # Validations
+  validates :name, presence: true
+  validates :phone, presence: true, :numericality => true 
+  validates :email, presence: true, :uniqueness => true
+  # Scopes
```

Comment:
@arps1989dev email can not `uniqness => true` because I can contact to admin many times.